query_optimizer/tests/execution_generator/Select.test

```diff
@@ -953,19 +953,79 @@ WHERE double_col < 0
 ==
 
 # Window Aggregation Test.
-# Currently this is not supported, an empty table will be returned.
-SELECT avg(int_col) OVER w FROM test
+SELECT char_col, long_col, avg(long_col) OVER w FROM test
 WINDOW w AS
-(PARTITION BY char_col
- ORDER BY long_col DESC NULLS LAST
+(ORDER BY char_col DESC NULLS LAST
```

Comment:
When the `RANGE` mode is introduced, a new parameter will be added to the `WindowAggregationHandle::calculate()` method, and would most likely be `const std::vector<attribute_id> &order_by_ids`. As for the cases where `ORDER BY` clause is missing, the vector would simply be empty, which indicates that the table is sorted by `tuple_id`. 

I don't think we need to explicitly add `tuple_id` as a sort attribute, because it is not actually an attribute.